12.0.Dockerfile

```diff
@@ -0,0 +1,165 @@
+FROM python:3.7-stretch AS base
+
+# Enable Odoo user and filestore
+RUN useradd -md /home/odoo -s /bin/false odoo \
+    && mkdir -p /var/lib/odoo \
+    && chown -R odoo:odoo /var/lib/odoo \
+    && sync
+VOLUME ["/var/lib/odoo"]
+EXPOSE 8069 8072
+
+ARG WKHTMLTOPDF_VERSION=0.12.5
+ARG WKHTMLTOPDF_CHECKSUM='1140b0ab02aa6e17346af2f14ed0de807376de475ba90e1db3975f112fbd20bb'
+ENV DB_FILTER=.* \
+    DEPTH_DEFAULT=1 \
+    DEPTH_MERGE=100 \
+    EMAIL=https://hub.docker.com/r/tecnativa/odoo \
+    GIT_AUTHOR_NAME=docker-odoo \
+    INITIAL_LANG="" \
+    LC_ALL=C.UTF-8 \
+    NODE_PATH=/usr/local/lib/node_modules:/usr/lib/node_modules \
+    OPENERP_SERVER=/opt/odoo/auto/odoo.conf \
+    PATH="/home/odoo/.local/bin:$PATH" \
+    PIP_NO_CACHE_DIR=0 \
+    PTVSD_ARGS="--server-host 0.0.0.0 --port 6899" \
+    PTVSD_ENABLE=0 \
+    PUDB_RDB_HOST=0.0.0.0 \
+    PUDB_RDB_PORT=6899 \
+    PYTHONOPTIMIZE=1 \
+    UNACCENT=true \
+    WAIT_DB=true \
+    WDB_NO_BROWSER_AUTO_OPEN=True \
+    WDB_SOCKET_SERVER=wdb \
+    WDB_WEB_PORT=1984 \
+    WDB_WEB_SERVER=localhost
+
+# Other requirements and recommendations to run Odoo
+# See https://github.com/$ODOO_SOURCE/blob/$ODOO_VERSION/debian/control
+RUN apt-get -qq update \
+    && apt-get -yqq upgrade \
+    && apt-get install -yqq --no-install-recommends \
+        chromium \
```

Comment:
I have needed saasc for making it work

Reply:
I think it's not needed if you install the python sass library. I want to test